index.html

```diff
@@ -25,6 +25,9 @@
         <nav class="navbar">
             <a class="name-container" href="logo">Bezalell</a>
             <img class="union" src="images/Union.png" alt="">
+            <div class="linkstome"> 
```

Comment:
well done Bazallel, the linters checked out fine, your code is professional and you used correct GitHub flow